src/components/FileTree.tsx

```diff
@@ -25,6 +26,9 @@ enum FileTreeViewEvent {
 const BATCHED_UPDATE_MAX_DEBOUNCE_MS = 4
 
 export class FileTree extends React.Component<IFileTreeProps> {
+	public static defaultProps = {
+		disableCache: false
+	 }
```

Comment:
We can remove this section as an `undefined` value will serve as falsy value anyway. It's not "bad" in any way, it's just that it'd be inconsistent since there are no `defaultProps` set up for other props.

Reply:
Hi @zikaari,
I have resolved the review comment and removed the defaultProps for from FileTree

Reply:
Hi @zikaari, any update on this?